site/content/entry/yuzu-progress-report-oct-2021/index.md

```diff
@@ -0,0 +1,430 @@
+- `CPU Video Decoding` means processing videos using the CPU, this can be useful for AMD Windows users, since the AMD driver is notorious for having bad VP9 video decoding 
+performance on Vega and newer GPU series, and using the CPU to render could be both faster and more accurate.
+- `GPU Video Decoding` means actually using the GPUs decoding capabilities to process videos, this is considerably faster than CPU decoding and is the default option. If a video 
+format is not compatible with the user’s GPU, yuzu’s NVDEC emulation will automatically fallback to CPU decoding.
+
+For some time we were wondering why the Switch had an open spot on their language selection.
+Turns out, with the release of `Mario Party Superstars`, Nintendo added support for Brazilian Portuguese!
+Morph [rushed to add support for it on yuzu](https://github.com/yuzu-emu/yuzu/pull/7244).
+Our game language selection menu can be found on `Emulation > Configure… > System > System > Language`.
+Jogo bonito on your gameplay sessions!
+
+The text on the TAS configuration window looked cramped, so [Behunin](https://github.com/behunin) edited the 
+[UI file and reformatted the text](https://github.com/yuzu-emu/yuzu/pull/7147) to make it easier to read.
+Later, [Moonlacer](https://github.com/Moonlacer) added to these changes by [fixing some grammar nits and adding an hyperlink](https://github.com/yuzu-emu/yuzu/pull/7197) that 
+links to the guide in our site.
+
+{{< single-title-imgs
+	" "
+	"tasbug.png"
+	"tasfix.png"
+>}}
+
+Moonlacer also [fixed a problem with the per-game configuration window](https://github.com/yuzu-emu/yuzu/pull/7223), where scaling the monitor over 100% would squash the 
+window.
+
+{{< single-title-imgs
+	" "
+	"csbug.png"
+	"csfix.png"
+>}}
+
+## Input changes
+
+When [german77](https://github.com/yuzu-emu/yuzu/pull/7142) added 
+[auto-centering support all the way back in July](https://yuzu-emu.org/entry/yuzu-progress-report-jul-2021/#input-changes), the range of the new center wasn’t considered, 
+making the analog sticks overpass 100% in a direction while failing to reach it in the opposite way.
+[Scaling the value of the range depending on the offset applied when centering](https://github.com/yuzu-emu/yuzu/pull/7142) solves this issue.
+
+## Future projects
+
+With the resolution scaler finished, our GPU devs got space to breathe, and they can focus back on work that was left for later.
+`Project Y.F.C` is an example.
+In Blinkhawk’s words, “there'll be Chicken Nuggets for everyone soon, a.k.a. it will cover many different things.”
+
+{{< imgs
+	"./wol.png|Lifelight intensifies (Super Smash Bros. Ultimate)"
+>}}
+
+`Project Gaia` is advancing very well, and `Project Kraken` [is out](https://github.com/yuzu-emu/yuzu/pull/7255) but didn’t make it in time for this progress report, so we 
+will focus on it next time.
+
+That’s all folks! Thank you so much for your kind attention, and see you next month!
```

Comment:
```suggestion
With the resolution scaler finished, our GPU devs have space to breathe, and they can focus back on work that was reserved for later.
`Project Y.F.C` is an example.
In Blinkhawk’s words, “there'll be Chicken Nuggets for everyone soon, a.k.a. it will cover many different things.”

{{< imgs
	"./wol.png|Lifelight intensifies (Super Smash Bros. Ultimate)"
>}}

`Project Gaia` is advancing very well, and `Project Kraken` (our Input Rewrite) [is out](https://github.com/yuzu-emu/yuzu/pull/7255) but didn’t make it in time for this progress report. Don't worry, we 
will discuss it next time!

That’s all folks! Thank you so much for your attention and see you next month!
```